symplyphysics/laws/electricity/ampere_law.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+
+# Description
+## Ampere's law is the law that determines the force with which a magnetic field acts on
+## a small segment of a conductor with a current. The force turns out to be linearly dependent 
+## on both current and magnetic induction.
+
+## Law is: F = I * L * B * cos(a), where
```

Comment:
Are there any requirements to the conductor geometry. Put them in conditions, if there are any.